packages/shared/src/style/reset.style.ts

```diff
@@ -1,7 +1,4 @@
-import { normalize } from 'polished';
-
 export const resetStyle = `
-${normalize()};
```

Comment:
@chybisov I just removed this. Do you see any issues here?

Reply:
Since nothing from resetStyle is used I'd remove the whole style folder

Reply:
Right, much cleaner now